cirq/ops/raw_types.py

```diff
@@ -262,21 +262,14 @@ def __pow__(self, power):
     def __call__(self, *args, **kwargs):
         return self.on(*args, **kwargs)
 
-    def controlled_by(self,
-                      *control_qubits: Qid,
-                      control_values: Optional[Sequence[
-                          Union[int, Collection[int]]]] = None) -> 'Gate':
+    def control(self,
+                control_values: Optional[Sequence[
+                    Union[int, Collection[int]]]] = None) -> 'Gate':
         """Returns a controlled version of this gate.
-
-        Args:
-            control_qubits: Optional qubits to control the gate by.
         """
         # Avoids circular import.
         from cirq.ops import ControlledGate
-        if len(control_qubits) == 0:
-            return self
```

Comment:
Change the condition to check the length of control_values.

Reply:
I think the default should be to create a ControlledGate with one control qubit.